databases/sqlitebrowser/Portfile

```diff
@@ -60,15 +62,29 @@ platform darwin {
     # Uses QSysInfo::buildAbi and QSysInfo::currentCpuArchitecture
     qt5.min_version     5.4
 
+    variant sqlite3 conflicts sqlcipher description {Use sqlite3 as db engine} {
+        depends_lib-append port:sqlite3
+    }
+
+    variant sqlcipher conflicts sqlite3 description {Use sqlcipher as db engine} {
+        depends_lib-append port:sqlcipher
+        configure.args-append CONFIG+=sqlcipher
+        # use `-Dsqlcipher=1' for cmake
+    }
+
+    if (![variant_isset sqlcipher]) {
+        default_variants  +sqlite3
+    }
+
     pre-configure {
         reinplace "s|libs/qscintilla/Qt4Qt5/qscintilla.pro||g" ${worksrcpath}/sqlitebrowser.pro
         reinplace "s|-lqscintilla2|-lqscintilla2_qt5|g" ${worksrcpath}/src/src.pro
         reinplace "s|\$\$PWD/../libs/qscintilla/Qt4Qt5||g" ${worksrcpath}/src/src.pro
         delete ${worksrcpath}/libs/qscintilla
     }
 
-    use_parallel_build  no
-    depends_lib-append  port:sqlite3
+    use_parallel_build  yes
```

Comment:
was the [original issue](https://trac.macports.org/ticket/60626) for why this was added resolved?